cli/sync_want.go

```diff
@@ -0,0 +1,45 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+
+	"golang.org/x/xerrors"
+
+	"github.com/coder/serpent"
+
+	"github.com/coder/coder/v2/agent/agentsocket"
+)
+
+func (r *RootCmd) syncWant() *serpent.Command {
+	return &serpent.Command{
+		Use:   "want <unit> <depends-on>",
+		Short: "Declare a dependency relationship",
+		Long:  "Declare that a service unit depends on another unit completing before it can start. This builds the dependency graph that coordinates service startup order. The dependent unit will not be ready to start until the dependency reaches complete status.",
```

Comment:
```suggestion
		Long:  "Declare that a this unit depends on another unit. This unit will not be ready to start until the other unit reaches the 'complete' status.",
```

Suggestion: Avoid complex language, make it clear which argument we're referring to by using the arguments name (inferable).